.gitignore

```diff
@@ -1,6 +1,3 @@
 node_modules
 dist
 dist_tmp
-.vscode
-*.d.ts
-*.tgz
```

Comment:
I can understand removing `.vscode`, if you want to share your editor's config, but we probably want to still ignore `*.d.ts` and `*.tgz` files.